packages/fast-components-react-base/src/radio/radio.props.ts

```diff
@@ -42,6 +42,6 @@ export interface RadioHandledProps extends RadioManagedClasses {
 }
 
 export interface RadioUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
-export type RadioOnChange = (event?: React.ChangeEvent<HTMLElement>) => void;
+export type RadioOnChange = (event?: React.ChangeEvent<HTMLDivElement>) => void;
```

Comment:
This isn't an input element?

Reply:
Yes...and this is why I went to sleep :)